### ISSUE TITLE
Tests fail on DRTS due to inability to reach timeserver

### ISSUE THREAD

**Issue by jweber-ice** [WIPACrepo/stf#109]
```
[deggtest@hydrometeor stf]$ STF_DEBUG=1 python3 tests/Flash.py -vvv
DEBUG >> <module> -> get_config -> __init__ -> _createConfig parsing config... /home/deggtest/jweber/stf/stfconfig.json
DEBUG >> <module> -> get_config -> __init__ -> _createConfig setting home to /home/deggtest/jweber/stf
DEBUG >> <module> -> get_config -> __init__ -> _createConfig parsing config... /home/deggtest/jweber/stf/stfconfig.json
DEBUG >> <module> -> get_config -> __init__ -> _createConfig parsing config... /home/deggtest/jweber/stf/stfconfig.local.json
DEBUG >> <module> -> get_config -> __init__ -> _createConfig setting home to /home/deggtest/jweber/stf
DEBUG >> <module> -> register -> get_run_args args: (Namespace(mbsnum=None, meta=None, metafile=None, testconfig=None), ['-vvv'])
DEBUG >> <module> -> run -> check_system_clock timesync=verify... checking WebAPI
DEBUG >> <module> -> run -> check_system_clock FAIL
Traceback (most recent call last):
  File "tests/Flash.py", line 63, in <module>
    stf.run()
  File "/home/deggtest/jweber/stf/stf/core.py", line 162, in run
    check_system_clock()
  File "/home/deggtest/jweber/stf/stf/core.py", line 204, in check_system_clock
    raise exceptions.STFRefuseToRun(msg)
stf.util.exceptions.STFRefuseToRun: System time is inaccurate. Cannot proceed
```
Issue occurs on both DRTS-1 and DRTS-2.
Oddly this happens most frequently when DRTS has not been used for a while. Re-running any test will then succeed.

**Comment by jweber-ice** [comment]
Consider increasing external access timeout in Python code. Consider adding command line argument to skip this check

**Comment by jweber-ice** [comment]
Check if time sync file in tmpdir is lost or affected

**Comment by jweber-ice** [comment]
Fixed in commit 82ff0ff820c05ca02df262ecb01f6f53222b3804
